inc/views/page_layout.php

```diff
@@ -18,7 +18,73 @@ class Page_Layout extends Base_View {
 	 * Init function
 	 */
 	public function init() {
-		add_action( 'neve_do_single_page', array( $this, 'render_page' ) );
+		add_action( 'neve_after_header_wrapper_hook', [ $this, 'render_cover_header' ] );
+		add_action( 'neve_do_single_page', [ $this, 'render_page' ] );
+		add_filter( 'neve_page_title_alignment_style', [ $this, 'filter_title_alignment_style' ], 10, 2 );
```

Comment:
can we maybe have a single filter that does this and maybe have an additional argument that will target post or page values?